launch-playground.sh

```diff
@@ -36,7 +37,16 @@ case "${1}" in
 esac
 
 cd ${playground_dir}
-docker-compose up ${components}
 
-# Clean Docker containers when you quit this script
-docker-compose down
+isComposeV1=`which docker-compose`
+if [ $? -eq 0 ]; then
+  docker-compose up ${components}
```

Comment:
I think we should prioritize `docker compose` since it's newer.

Reply:
I agree with @ryankert01 , since recent version of docker shipped with `docker compose`, it's better to use recent version of docker to eliminate potential bugs.